### ISSUE TITLE
Missing tube length both in the data model and in the digital calibration certificate

### ISSUE THREAD

**Issue by kersting** [IEA-Task-43/digital_wra_data_standard#151]
The IEC 61400-12-2 requires the length of the tube used during the anemometer calibration to verify whether if the installation of the anemometer is in line with the guideline's requirements. As a result, that piece of information needs to be added both in the data model and in the digital calibration certificate.

**Comment by stephenholleran** [comment]
@heikowestermann what do you think we could do here? It seems like we should be including it.

**Comment by heikowestermann** [comment]
Until now my checklist was the IEC 61400-12-1, but I am happy to extend the schema to the 12-2. I already discussed with @kersting that we could add that information. Having said that, I don't know how many calibration labs have that information readily available as at least MEASNET is only based upon the 12-1. I would suggest making the information optional, as opposed to required to not hinder adoption from the labs.

**Comment by stephenholleran** [comment]
Sounds like a plan; add it but make it optional.

**Comment by kersting** [comment]
@heikowestermann and @stephenholleran thanks for looking into this. Please let me know if I can help either modifying the model or reviewing the modifications.

**Comment by stephenholleran** [comment]
From our discussion https://github.com/IEA-Task-43/digital_wra_data_standard/discussions/129#discussioncomment-2431888

> HW thinks it is a good thing to add to the DCC. GK will share an image showing this requirement. 
> The tube length during calibration will be different to that used in installation but the installed one needs to be the same length or longer than the calibration one.

**Comment by stephenholleran** [comment]
This is now implemented into the DCC as `mounting_length`.

It now needs to be implemented in the WRA Data Model.

**Comment by kersting** [comment]
@stephenholleran do you need my support to get this into the WRA data model?

**Comment by stephenholleran** [comment]
@kersting that would be great if you could have a go at this. Thanks.

**Comment by kersting** [comment]
@stephenholleran should we keep the same naming as for the digital calibration certificate? There we have `mounting_lenght`. IEC references it as just tube so I was thinking about using `tube_mounting_lenght`. Thoughts?

**Comment by stephenholleran** [comment]
@kersting
I see your point. What about `mounting_tube_length_mm`?

In the description we should say something like:
> The length, in millimeters, of the tube that the sensor is directly mounted on. This is used if the mounting tube is different to the `upstand_height_mm` as in some cases the total `upstand_height_mm` may be made up of more than one tube. This parameter may be used to satisfy the condition in the IEC 61400-12-1:2017 that the **_"Tube length ≥ tube length used during calibration"_** where the **_"tube length used during calibration"_** relates to the `mounting_length` in the digital calibration certificate.

Suggestions welcome?

---

This then begs the question; what is the diameter of the tube the sensor is mounted on? I would use the `upstand_diameter_mm` in this case even though there is another tube with a larger diameter as the one directly mounting the sensor is the important one.

**Comment by kersting** [comment]
@stephenholleran I agree with your proposal and observation about the `usptand_diameter_mm`. I'm happy to make the additions for both in the data model and make a pull request. Let me know if I can proceed.

**Comment by stephenholleran** [comment]
@kersting feel free to go for it.

**Comment by kersting** [comment]
@stephenholleran a pull request has been made. Please take a look on https://github.com/IEA-Task-43/digital_wra_data_standard/pull/187
